compiler/testData/debug/stepping/smap.kt

```diff
@@ -0,0 +1,63 @@
+
```

Comment:
Name of test a bit confusing. Could you rename it?

Reply:
Sure, renamed. Also left a comment to indicate this is from `.../testData/codegen/boxInline/smap/smap.kt`